.gitignore

```diff
@@ -5,3 +5,6 @@ venv/
 *.pyc
 example_pgen.rs
 js_parser/parser_tables.py
+jsparagus/emit/collect_handler_info/info.json
+jsparagus/emit/collect_handler_info/target
+*~
```

Comment:
I think `*~` belongs in `$HOME/.gitignore` for users who need it. The other two are all right, though.

Reply:
I didn't know that there's global configuration, thanks!